Makefile

```diff
@@ -255,6 +255,7 @@ modules-download: ## Downloads and caches the modules
 .PHONY: generate
 generate: ## Generate code
 	$(MAKE) generate-go
+	$(MAKE) generate-go-conversions
```

Comment:
This had been purposely omitted from here for the time being because it is just so slow (~75sec on my fast laptop), and I didn't want to impact people's existing workflow. I'd prefer we make that target not PHONY before we enable it by default.

Reply:
Removed this one.